### DISCUSSION TITLE
Sugestion for manual

### DISCUSSION THREAD

**Discussion by Sailor-Fhem** [niccokunzmann/open-web-calendar#502] [Ideas]
My sugestion for amending the manual:
[https://open-web-calendar.quelltext.eu/host/pypi/](url)

# Python Package

![](https://img.shields.io/pypi/v/open-web-calendar.svg) ![](https://img.shields.io/pypi/pyversions/open-web-calendar.svg)

The Open Web Calendar is available as a Python package on [PyPI]({{link.pypi}}).
If you have [Python 3](https://www.python.org/) installed, run this to install the package:

```shell
pip install open-web-calendar
```

At this point you might get the error message
```shell
error: externally-managed-environment
```
On your productive system, it is strongly recommended to follow the instructions to create a virtual environment using python3 -m venv path/to/venv.

After installation, run the Open Web Calendar using [Gunicorn](https://pypi.org/project/gunicorn/):

```shell
gunicorn open_web_calendar:app
```

You should now see the server running at [http://127.0.0.1:8000](http://127.0.0.1:8000).

However, in case you have installed the WebServer you are going to use for the integration of this calendar on the same local machine, it must be accessible other computer as well. In this case you might want to start the open_web_calendar with the dedicated IP-address.

```shell
gunicorn -b 192.168.178.7 open_web_calendar:app
```
You should now see the server running at [http://192.168.178.7:8000](http://192.168.178.7:8000) not only from your local machine.

For more configuration options, see here:

- [Gunicorn](https://pypi.org/project/gunicorn/)
- [Configuration](../configure)

```

**Comment by niccokunzmann** [comment]
This looks really good! Could you create a pull request with this?
Maybe, you can find a link that explains the venv for newcomers, too. A pull request with or without such a link would be much apprecated!

**Reply by niccokunzmann** [reply]
If you do that, you can participate in the hacktoberfest: https://hacktoberfest.com/

**Comment by Sailor-Fhem** [comment]
Done
